data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+			`	 Landing in the hangar designated for your fleet doesn't do the Syndicate's image any favors, either. Workers are actively repairing parts of the walls and ceiling that you felt should've been replaced long ago, and the rickety scaffolding they've erected for the repair crews makes you worried, to say the least. The air in the hangar is also thick and stagnant, and almost makes you gag. You quickly call over the captains of the auxiliaries and make your way to the loading docks - it takes about ten minutes to make your way through the maze of corridors before you reach your destination.`
+			`	Upon reaching the docks, several Syndicate dockworkers call you over to a large collection of cargo containers filled with the building materials bound for Geminus. "Took you long enough," one says with a cigarette hanging out the side of his mouth. "We've been waiting here for five hours with our thumbs up our collective asses for you Navy jarheads. One of your own came by here about a month ago and acted like they owned the place - I tell ya, my boss doesn't pay me enough to put up with shit like theirs."`
+			choice
+				`	"Sorry to have kept you waiting. We came as quickly as we could."`
+					goto end
+				`	"You better keep that tongue of yours in check, my man, or you'll be choking on that cigarette."`
+					goto fight
+				`	"Hold on. 'One of our own?' You mean someone from the Navy?"`
+					goto investigate
+			label fight
+			apply
+				set "bad look"
+			`The dockworker chuckles for a second, takes his cigarette out of his mouth and flicks it at you. You almost throw a punch before one of the captains of the auxiliaries steps between the two of you. "Sir, it's been a long day for all of us. Could you please help us load our cargo onto our ships?"`
+			`"Yeah, yeah, we'll get on it," he replies. "And thanks for proving my point, you jumpy little shit," he pointedly directs at you. Only then do you realize Danforth will probably hear of your behavior - and it's a sure bet he will not be pleased by your conduct.
+				goto end
+			label investigate
+			`"Who the FUCK did you think I meant, asshole?!" he almost screams at you as the cigarette falls out of his mouth. "You know goddamn well I meant someone from the Navy! I knew you fuckers were uptight, but don't tell me you're all a bunch of fucking idiots, too!"`
```

Comment:
This is a rather extreme reaction for a dockworker, I doubt most of these people are too seriously put-out by getting paid to *not* do work for a few hours of waiting.